_foundations/eclipse.md

```diff
@@ -42,17 +41,17 @@ finance:
   sponsorList: "CA Technologies, CEA LIST, Ericcson AB, IBM, itemis AG, OBEO, Oracle, Redhat, Bosch, SAP"
 brand:
   primary: "Eclipse"
-  secondary: "various projects - dozens+"
+  secondary: "LocationTech, PolarSys, various projects - dozens+"
   registered: "yes"
   logo: "https://www.eclipse.org/eclipse.org-common/themes/solstice/public/images/logo/eclipse-426x100.png"
-  logoRegistered: "no"
+  logoRegistered: "yes"
   policyURL: "https://eclipse.org/legal/logo_guidelines.php"
   useURL: ""
   comments: "Detailed rules for using Eclipse and all associated project trademarks."
 projects:
-  total: "84"
+  total: "345"
```

Comment:
Question: how did you count projects?
The Apache definition is Top Level Projects run by a mature PMC - not including incubator, attic (dormant), labs (individual committers or small codebases that aren't a full project), and not including subprojects (for example, the Apache number does not include Solr, because it's a subproject of the Lucene PMC).
Can you get me a number that matches that criteria for Eclipse?  I'm thinking it should be closer to 84, and not 345. 8-)

Reply:
The Eclipse Foundation's definition of "top level" and "subproject" are different from Apache's. We also don't have a notion of labs (other than, perhaps, permanent incubators or de facto components within projects). If I remove the permanent incubators (and account for a recent collapse of the Eclipse JDT project hierarchy), we end up with 325. I'll use that number.

Reply:
Actually, if I further prune out the mid-level container projects that that don't host actual development, I can whittle the number down to 283.